cql/DashboardLibrary.cql

```diff
@@ -162,6 +162,9 @@ define function CytologyInterpretation(conclusions List<System.Concept>):
         when c ~ "ASC-H" then {text: 'ASC-H', rank: 5}
         when c ~ "AGC" then {text: 'AGC', rank: 6}
         when c in "HSIL" or c ~ "SCC" then {text: 'HSIL+', rank: 7}
+        when c ~ "AGC Favor Neoplasia" then {text: 'AGC-FN', rank: 8}
+        when c ~ "Endocervical Cells Favor Neoplasia" then {text: 'AEC-FN', rank: 8}
```

Comment:
`HighestRankedInterpretation` will not know how to rank cytology results that have the same rank. If both exist in the results section, it will most likely just choose one. How do we want a result that may have both of these to display to the front end? (see FormatReports function)

Reply:
I don't know. I don't think I'm the person to answer that. But I don't think it's impossible.

I will say that both of those results would lead to the same recommendation - they are combined via "OR" in the following G1.5 recommendation: https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/main/cql/ManageRareAbnormality.cql#L413. So it may not be the biggest deal that they have the same rank, especially considering we plan to re-visit how we prioritize/rank rare abnormality results after we discuss with clinical teams.

Reply:
Can you please open 1 or 2 Jira tickets if these will be included in separate PR(s)?

Reply:
See [CCSMCDS-137](https://jira.mitre.org/browse/CCSMCDS-137) - update rare cytology logic to respect rankings
[CCSMCDS-141](https://jira.mitre.org/browse/CCSMCDS-141) - Update patient history to include all results from tests